packages/next/build/webpack-config.ts

```diff
@@ -476,7 +476,7 @@ export default async function getBaseWebpackConfig(
             // Default pages have to be transpiled
             if (
               !res.match(/next[/\\]dist[/\\]next-server[/\\]/) &&
```

Comment:
```suggestion
              !res.match(/[/\\]next[/\\]dist[/\\]next-server[/\\]/) &&
```

Reply:
This looks a folder of `next-server` as well so it not applies to other libs :]